doc/user/content/release-notes.md

```diff
@@ -107,6 +107,22 @@ These changes are present in [unstable builds](/versions/#unstable-builds) and
 are slated for inclusion in the next stable release. There may be additional
 changes that have not yet been documented.
 
+- **Breaking change.** Store days separately in [`interval`](/sql/types/interval). 
+  Unlike the previous version, Hours are never converted to days. This
+  means that an Interval of 24 hours will not be equal to an Interval 
+  of 1 day, you cannot subtract hours from days, and when ordering
+  Intervals d days > h hours for all d,h {{% gh 10708 %}}.
```

Comment:
I'm still trying to figure out if it's possible to properly order Interval's by converting 24 hrs to 1 day without doing the same thing with equality. The docs for the `PartialOrd` trait say
> The methods of this trait must be consistent with each other and with those of PartialEq in the following sense:
a == b if and only if partial_cmp(a, b) == Some(Equal).
...

https://doc.rust-lang.org/std/cmp/trait.PartialOrd.html

Reply:
Do we have a similar behavior with months, where any number of months is greater than any number of days? Can you add some tests for these to assert our incorrect behavior with a comment linking to some github issue about it?

Reply:
The behavior is the same with months, it's just that behavior already existed so I didn't include it in the release notes. I'll add some tests and include a GH link.